README.md

```diff
@@ -183,7 +182,7 @@ For a good example package, see [base64-js](https://github.com/DefinitelyTyped/D
 #### Common mistakes
 
 * First, follow advice from the [handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html).
-* Formatting: Use 4 spaces. For new code, this is enforced by Prettier.
+* Formatting: Use 4 spaces. Prettier is set up on this repo, so you can run `npm run prettier -- --write path/to/package`.
```

Comment:
:+1:

Now that we support only 2.8 upward, prettier has a fighting chance on the whole repo. Maybe next tsconf?

Reply:
👍